test/app-tests/loading.js

```diff
@@ -47,7 +47,7 @@ describe('loading:', function () {
     // the mounted MatrixChat
     let matrixChat;
 
-    // a promise which resolves when the MatrixChat calls onLoadCompleted
+    // a promise which resolves when the MatrixChat calls onTokenLoginCompleted
```

Comment:
But it only calls onTokenLoginCompleted if the token login was a success. Should this test not just fail? Unless it logins in with a token explicitly?